.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+### Cmdlet(s)
```

Comment:
(Please include the exact command invoication with parameters that is causing the issue)